nuxeo-distribution/nuxeo-nxr-server/pom.xml

```diff
@@ -654,6 +654,15 @@
       <artifactId>quartz-mongodb</artifactId>
     </dependency>
 
+    <dependency>
```

Comment:
Please add a comment explaining that this isn't needed by the platform but is a useful log4j appender in containers.